t/t3438-rebase-git-logs-symlink.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+test_description='git rebase when .git/logs is a symlink
```

Comment:
This is an awfully specific test in a sprawling test suite. Could we make this a bit more generic, or even better: could we put this into an existing `t34*` test script?

Reply:
I only read `t/README` and I didn't read the "Add regressions tests" wiki page so I missed the information that it was better to add a test case in an existing test, sorry for this.
I created a test case in `t3400-rebase.sh` instead.